### PR TITLE
[Docs] Update quick start v2 related docs info

### PR DESCRIPTION
**Pull request by tcodehuber**
### Purpose of this pull request
As titiled

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by Hisoka-X** [comment]
Thanks @tcodehuber for created this PR! Could you follow the guide to open github action on your fork repository? https://github.com/apache/seatunnel/pull/7161/checks?check_run_id=27280340813

**Comment by tcodehuber** [comment]
> Thanks @tcodehuber for created this PR! Could you follow the guide to open github action on your fork repository? https://github.com/apache/seatunnel/pull/7161/checks?check_run_id=27280340813

@Hisoka-X Everything is OK, PLAL.

File: docs/en/start-v2/locally/deployment.md [hunk 1]
```diff
@@ -61,14 +61,9 @@ You can find all supported connectors and the corresponding plugin_config config
...
 
 :::
 
 Now you have completed the download of the SeaTunnel installation package and the download of the connector plug-in. Next, you can choose different running modes according to your needs to run or deploy SeaTunnel.
 
-If you use Flink to run the synchronization task, in this mode, there is no need to deploy the SeaTunnel Engine service cluster. You can refer to [Quick Start of SeaTunnel Flink Engine](quick-start-flink.md) to run your synchronization task.
-
-If you use Spark to run the synchronization task, in this mode, there is no need to deploy the SeaTunnel Engine service cluster. You can refer to [Quick Start of SeaTunnel Spark Engine](quick-start-spark.md) to run your synchronization task.
-
-If you use the SeaTunnel Engine (Zeta) that comes with SeaTunnel to run tasks, you need to deploy the SeaTunnel Engine service first. Refer to [Deployment of SeaTunnel Engine (Zeta) Service](quick-start-seatunnel-engine.md).
```
Comment (Hisoka-X) [inline docs/en/start-v2/locally/deployment.md]: why removed this part?

Comment (tcodehuber) [reply docs/en/start-v2/locally/deployment.md]: @Hisoka-X  Sorry ,it's my fault. Updated again.

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @tcodehuber !

**Review by hailin0 — APPROVED** [review]

**Pull request merged**